common/app/views/support/HtmlCleaner.scala

```diff
@@ -103,7 +103,7 @@ case class PictureCleaner(article: Article, amp: Boolean)(implicit request: Requ
       container <- findContainerFromId(figure.attr("data-media-id"), image.attr("src"))
       image <- container.largestImage
     }{
-      val hinting = findBreakpointWidths(figure)
+      val hinting = findBreakpointWidths(figure, article.isImmersive)
```

Comment:
Hmmn, this is nearly consistent, but look below at `LiveBlogMedia`. We choose the `ContentRelation` object based on the content type. So if we had a `ImmersiveMedia`, alongside `LiveBlogMedia` and `BodyMedia`, then there would be no `isImmersive` flags needed. It's also more scalable, when we inevitably introduce `SuperDuperImmersiveMedia`